### DISCUSSION TITLE
v4.6.1: Updates on maintenance

### DISCUSSION THREAD

**Discussion by orazioedoardo** [pivpn/pivpn#1832] [Announcements]
Hey there, after the [previous release](https://github.com/pivpn/pivpn/releases/tag/v4.6.0), there have been questions about the future of PiVPN. First off, I would like to thank you all for the appreciation comments. Second, users that have previously installed PiVPN will be able to continue using it by simply updating the system via the package manager, or just periodically reboot if you have opted-in to unattended-upgrades.

Issues and pull requests have been reopened but will be moderated a bit more: Issues that do not include a [debug log](https://docs.pivpn.io/openvpn/#help-with-troubleshooting) (unless not applicable) will be closed immediately. Connectivity issues MUST show effort in following [troubleshooting steps](https://docs.pivpn.io/faq/#how-do-i-troubleshoot-connection-issues) otherwise they will also be closed immediately. Use discussions for general... discussions that are not issues with the scripts themselves.

**PiVPN will continue to be maintained on a best-effort basis by me, I will update the script to keep it working on future release of distributions but large efforts are less likely unless someone opens a pull request for the feature (in this case, please add me as reviewer).**

**Full Changelog**: https://github.com/pivpn/pivpn/compare/v4.6.0...v4.6.1

**Comment by jook-network** [comment]
I'm a bit confused. On pivpn.io there still is a message saying PiVPN is no langer being maintained.

**Reply by orazioedoardo** [reply]
> On pivpn.io there still is a message saying PiVPN is no langer being maintained.

Oops, forgot to remove that sentence there.

**Comment by ddubdub** [comment]
Hi there, 

thanks a lot for keeping it up. Really appreciate your efforts
